x-pack/legacy/plugins/siem/public/components/events_viewer/events_viewer.test.tsx

```diff
@@ -20,8 +20,17 @@ jest.mock('../../lib/settings/use_kibana_ui_setting');
 
 const from = 1566943856794;
 const to = 1566857456791;
-
+// Suppress warnings about "act" until async/await syntax is supported: https://github.com/facebook/react/issues/14769
```

Comment:
we might need to do something global instead of doing it for every single test files

Reply:
https://github.com/elastic/kibana/issues/46469